editor/src/components/canvas/canvas-strategies/keyboard-absolute-move-strategy.ts

```diff
@@ -46,56 +46,49 @@ export const keyboardAbsoluteMoveStrategy: CanvasStrategy = {
     return 0
   },
   apply: (canvasState, interactionState, sessionState) => {
-    // TODO: absolutely minimal implementation
     if (interactionState.interactionData.type === 'KEYBOARD') {
       const key = interactionState.interactionData.keysPressed[0]
```

Comment:
I believe this should be mapping over the `keysPressed`, so that it can handle e.g. up + left being held down